tests/utils.go

```diff
@@ -1953,60 +1952,14 @@ func RenderPod(name string, cmd []string, args []string) *k8sv1.Pod {
 	return &pod
 }
 
-func RenderPodWithPVC(name string, cmd []string, args []string, pvc *k8sv1.PersistentVolumeClaim) *k8sv1.Pod {
-	pod := RenderPod(name, cmd, args)
```

Comment:
Is there a rationale for no longer using RenderPod?
I assume it's reasonable to have libstorage use utils and we are breaking up utils to logical divisions.

Reply:
That makes sense then :)
If we are considering moving RenderPod to a libpod then it might make sense to have an /* XXX use renderpod once it is out of utils.go */ so we remember to use it.